pkg/game/game.go

```diff
@@ -161,6 +167,20 @@ func (game *Game) CurrentPlayer() elements.Player {
 	return game.players[game.currentPlayer]
 }
 
+func (game *Game) GetPlayerByID(playerID elements.ID) elements.Player {
+	for _, player := range game.players {
+		if player.ID() == playerID {
+			return player
+		}
+	}
```

Comment:
There's no need to loop here, just index by `playerID - 1`. You'll even get the panic for free.

Reply:
changed. I believe I did it that way, cause player could have any ID, when created with New(id elements.ID) elements.Player, and I wanted to make sure, it will be found :P